### ISSUE TITLE
[BUG] Cannot read property 'getFileName' of undefined

### ISSUE THREAD

**Issue by jimmywarting** [npm/write-file-atomic#85]
Sry, but the issue template needs to change... too much weird stuff to fill in.

I tried downloading the source and running `npm i && npm run test` from scratch.  But it fails

```
jimmy.warting@tr-c02zj2p learning-atomic % npm run test

> write-file-atomic@3.0.3 test
> tap

TypeError: Cannot read property 'getFileName' of undefined
    at module.exports (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/node_modules/caller-path/index.js:4:40)
    at importJSX (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/node_modules/import-jsx/index.js:24:46)
    at module.exports (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/node_modules/treport/lib/index.js:13:18)
    at exports.makeReporter (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/bin/run.js:422:23)
    at runTests (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/bin/run.js:744:3)
    at mainAsync (/Users/jimmy.warting/git/learning-atomic/node_modules/tap/bin/run.js:244:5)
TAP version 13
```

fyi, i use node v16.6.2

**Comment by jimmywarting** [comment]
think this might be the issue: https://github.com/sindresorhus/caller-path/pull/9

should update to caller-path@3

**Comment by wraithgar** [comment]
This should be fixed by #87
